### PR TITLE
Documentation update for v5.4.0 new features

### PR DESCRIPTION
**Pull request by venkateshjayagopal**
Documentation update for v5.4.0 for
1. Scan caching feature
2. Admission control rules design change
3. Adding CTRL_SEARCH_REGISTRIES env variable for admission control
4. Internal certificate rotation feature.
5. Change Zero Drift design change for v5.4.0
6. Added BusyBox process profile rule behavior with v5.4 and v5.3

### CODE REVIEW

**Review by jayhuang-suse — APPROVED** [review]

**Review by jayhuang-suse — APPROVED** [review]

File: versioned_docs/version-5.4/03.configuration/01.console/03.certrotate/03.certrotate.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Replacing Self-Signed Certificate
```
Comment (sunilarjun) [inline versioned_docs/version-5.4/03.configuration/01.console/03.certrotate/03.certrotate.md:2]:
```suggestion
title: Rotating Self-Signed Certificate
```

File: versioned_docs/version-5.4/05.policy/03.admission/03.admission.md [hunk 2]
```diff
@@ -255,6 +269,20 @@ The primary (master) cluster can scan a registry/repo designated as a federated
...
+
+```yaml
+containers:
+      - env:
+        - name: CTRL_SEARCH_REGISTRIES
+          value: https://recovery.quay.io/,https://aws.repository.com/
+```
+
+>>> NOTE: Please note, NeuVector doesn't tell Kubernetes about the default registry server when it's not specified in the YAML. NeuVector only uses the configured CTRL_SEARCH_REGISTRIES value for evaluating the admission control requests.
+Kubernetes still pulls image from the default registry server (configured in its runtime configuration) when it's not specified in the YAML. It means when registry server is not specified in the YAML, NeuVector & Kubernetes could use different default registry server URLs.
```
Comment (sunilarjun) [inline versioned_docs/version-5.4/05.policy/03.admission/03.admission.md:284]:
```suggestion
:::note
Please note, NeuVector doesn't tell Kubernetes about the default registry server when it's not specified in the YAML. NeuVector only uses the configured CTRL_SEARCH_REGISTRIES value for evaluating the admission control requests.
Kubernetes still pulls image from the default registry server (configured in its runtime configuration) when it's not specified in the YAML. It means when registry server is not specified in the YAML, NeuVector & Kubernetes could use different default registry server URLs.
:::
```

File: versioned_docs/version-5.4/06.scanning/07.scan_caching/scan_caching.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
+---
+title: Scan caching
+taxonomy:
+    category: docs
+slug: /scanning/scan_caching
+---
+
+> Note This feature is only supported from v5.4.0 onwards.
```
Comment (sunilarjun) [inline versioned_docs/version-5.4/06.scanning/07.scan_caching/scan_caching.md:8]:
```suggestion
:::note
This feature is only supported from v5.4.0 onwards.
:::
```

File: versioned_docs/version-5.4/06.scanning/07.scan_caching/scan_caching.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+      volumes:
+      - emptyDir:
+          sizeLimit: 50Mi
+        name: internal-cert-dir
+	  - hostPath
+		  path: /tmp/
+		  type: ""
+		name: scan-cache
+```
+> NOTE: Mounting ```/tmp/images/caches``` on PVC is not supported.
```
Comment (sunilarjun) [inline versioned_docs/version-5.4/06.scanning/07.scan_caching/scan_caching.md:181]:
```suggestion
:::note
Mounting ```/tmp/images/caches``` on PVC is not supported.
:::
```

**Review by sunilarjun — CHANGES_REQUESTED** [review]
Hello! Requesting some changes to the title from "Replacing" to "Rotating" and updating the note syntax to the Docusaurus admonition, otherwise looks good. Thank you!

**Comment by venkateshjayagopal** [comment]
Thanks for reviewing the changes. I have accepted your changes.

**Review by sunilarjun — APPROVED** [review]
Thanks for addressing the changes, approved!

**Pull request merged**